apps/s_client.c

```diff
@@ -3827,18 +3827,15 @@ static int user_data_process(struct user_data_st *user_data, size_t *len,
         char *cmd_start = buf_start;
 
         cmd_start[outlen] = '\0';
-        do {
-            cmd_start = strstr(cmd_start, "{");
-            if (cmd_start == buf_start && *(cmd_start + 1) == '{') {
-                /* The "{" is escaped, so skip it */
-                cmd_start += 2;
-                buf_start++;
-                user_data->bufoff++;
-                user_data->buflen--;
-                outlen--;
-                continue;
```

Comment:
Coverity was complaining about this `continue`:

> CID 1524619 (#1 of 1): Unexpected control flow (UNEXPECTED_CONTROL_FLOW)
> continue_in_do_while_false: A continue statement within a do ... while (...) loop only continues execution of the loop body if the loop continuation condition is still true. Since the condition will never be true in a do ... while (false) loop, the continue statement has the same effect as a break statement. Did you intend execution to continue at the top of the loop?

Which is a valid complaint IMO.  I just refactors the code a little in light of this but retained the behaviour.